scripts/validate_diff_accounts.sh

```
@@ -5,8 +5,7 @@
 # have corresponding account configuration files.
 
 # Get the list of changed files
-changed_files=$(git diff --cached --name-only)
-
+changed_files=${git diff --name-only origin/develop}
```

Shellcheck (0.10.0)

[warning] git is referenced but not assigned.

(SC2154)